src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleSearch = (value: string) => setQuery(value);
```

Comment:
How this function help you?

Reply:
> How this function help you?

@SanyaBratashchuk it's just for convenience. Should I delete it and make all changes via "onChange" directly?

Reply:
It is better to create handler which accept an event and set its target's value to the state